permissions/Mozilla.permission

```diff
@@ -1,8 +1,10 @@
 # -*- mode: sh -*-
 
 # x-sailjail-translation-catalog = sailjail-permissions
-# x-sailjail-translation-key = permission-la-webview
-# x-sailjail-description = Access webview
+# x-sailjail-translation-key-description = permission-la-webview
+# x-sailjail-description = Use webview
```

Comment:
Maybe "Browsing web" instead of webview.

Reply:
Although in context of email "browsing web" doesn't sound good.

Reply:
View Web Pages maybe? or Display

Reply:
Or "Web content" and long description telling about "Viewing web content". Idea is that longer description has those additional bits.

Reply:
Web content sounds good to me. Thanks!